### PR TITLE
Javadocs and Use google StringUtils for Nullchecks

### PR DESCRIPTION
**Pull request by Narimm**
Fix and Correct JavaDoc Errors that prevent a release build
Also in one case use StringUtils do perform a string null/empty check.

### CODE REVIEW

**Comment by RoyCurtis** [comment]
Thanks for correcting these. Before I pull; can you please explain what you mean by "prevent a release build"? I have been able to build GPrism fine without javadoc errors

File: src/main/java/me/botsko/prism/actions/BlockAction.java [hunk 1]
```diff
@@ -410,15 +411,15 @@ protected ChangeResult placeBlock(Player player, QueryParameters parameters, boo
                 final Skull skull = (Skull) block.getState();
                 skull.setRotation( s.getRotation() );
                 skull.setSkullType( s.getSkullType() );
-                if( s.owner != null && !s.owner.isEmpty() ) {
+                if( !Strings.isNullOrEmpty(s.owner) ) {
```
Comment (mikroskeem) [inline src/main/java/me/botsko/prism/actions/BlockAction.java:414]: But why? `Strings#isNullOrEmpty` does literally the same thing as the code you replaced. I find this change unneeded.

@RoyCurtis your opinion?

Comment (RoyCurtis) [reply src/main/java/me/botsko/prism/actions/BlockAction.java:414]: I disagree; I'd do the same thing. Subjectively speaking, I find it cleaner to read and understand than two separate operations. But not a big deal for me really :P

**Comment by Narimm** [comment]
Ok so when I use Maven release it by default will build javadocs....if your javadocs have errors the build will fail...so either you remove the javadocs or make sure they are correct.  Regarding the other change its minor I dont really mind if you exclude it...  We have been maintaining a prism for for over 2 years (since it was semi abandoned but we have so many others to look after someone else taking this one on is a godsend...merging in elixir was great as well

**Comment by Narimm** [comment]
There are a number of other changes on our branch I have yet to PR...I still need to assess whether we need them in light of the change you have made.

**Comment by RoyCurtis** [comment]
Thank you for explaining; I am still learning about Maven, so I did not know about `mvn release:prepare`. Hopefully, pulling in these changes will fix the issue for you.

I'd be curious to see the changes you guys have made. Are they the same ones mentioned here? https://github.com/Gamealition/GPrism/commit/d9b743cb9948c64a12f75c97e1001e8b2d757944#commitcomment-22224675

**Comment by Narimm** [comment]
If you really want to see how I do maven take a look at our current POM we
have an Automatic Build server - that auto builds - and copies the build into  a directory for inclusion on our server network...it also will perform maven release's that update...but you need to use the SNAPSHOT versioning system that maven suports ie 1.0.0-SNAPHOT which is MAJOR.MINOR.INCREMENTAL-QUALIFIER

**Comment by Narimm** [comment]
Roy I will try and PR any changes from our branch that are relevant to you...I actually think some are probably not....there were a few changes that I thought might be helpful I will do a compare and see what I get

**Comment by Narimm** [comment]
Just a heads up I completed a code review ...
basically the only change of consequence is that the legacy itemstack format you had never included bannermeta  ....given the migration away I doubt pulling that change is of any value to Gamealition.  Aside from that the only other changes are language level stuff ie using <> v <String> for  map declarations etc...ie no consequence.

**Comment by RoyCurtis** [comment]
Sorry for the slow response; thanks for checking and comparing!

Yes, with item stacks, I decided to just use native Bukkit serialization. As bulky as that might be, I wanted something more maintainable.

I wouldn't mind pulls of just syntax changes, if they make the code cleaner :)

**Comment by Narimm** [comment]
Sure I can look at making a syntax pull no probs

**Pull request merged**